tools/mpy_ld.py

```diff
@@ -520,6 +525,8 @@ def populate_got(env):
             dest = int(got_entry.name.split("+")[1], 16) // env.arch.word_size
         elif got_entry.sec_name == ".external.mp_fun_table":
             dest = got_entry.sym.mp_fun_table_offset
+        elif got_entry.sec_name == ".external.fixed_addr":
+            dest = ".fixed_addr"
```

Comment:
I think this could simply be:
```py
elif got_entry.sec_name == ".external.fixed_addr":
    continue
```
Then there doesn't need to be a special 126 code.

Reply:
Yes, you're right.  For some reason I kept thinking the relocation table and the relocation information were interleaved somehow, which is funny since I've been staring at MPY's hex dumps and their disassembled output for quite some time now so I should have known better!

I've tested your suggestion on an ESP8266 for one of the previously failing tests (`framebuf_polygon`) and it works.  I'll rebuild the PR without the extra relocation type in.